### PR TITLE
Dockerfile: bump Rust to toolchain 1.50.0 

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: bitbox-bridge/release/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,9 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends \
     cmake \
     curl \
     wget \
-    software-properties-common 
+    software-properties-common \
+    apt-transport-https \
+    ca-certificates
```
Comment (x1ddos) [inline bitbox-bridge/release/Dockerfile:14]: I would expect ca-certificates to be installed as a dependency on apt-transport-https:

image

https://packages.ubuntu.com/xenial/apt-transport-https

**Review by x1ddos — APPROVED** [review]
LGTM

File: bitbox-bridge/release/Dockerfile (hunk 1 above)
Comment (benma) [reply bitbox-bridge/release/Dockerfile:14]: Does `recommends` means it is gonna be installed?

Comment (benma) [reply bitbox-bridge/release/Dockerfile:14]: Docs seem unclear, but based on the wording I'd say better add explicitly.

Comment (x1ddos) [reply bitbox-bridge/release/Dockerfile:14]: :+1:

**Pull request merged**